content/docs/command-reference/add.md

```diff
@@ -243,3 +243,36 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+To untrack a file or directory just add
+[patterns](https://git-scm.com/docs/gitignore) (corresponding to the location of
+file or directory) under `.dvcignore` file.
+
+Let's create a sample script to understand this fact.
+
+```dvc
+$ echo test_v1 >> test1.txt
+$ echo test_v1_1 >> test2.txt
+
+$ dvc add test*
```

Comment:
How about more general terms and file names here? Something like
```suggestion
$ echo foo > foo.txt
$ echo bar > bar.txt
$ dvc add foo.txt bar.txt
```
Also single `>` are enough for this example, I think. Also, no need for that extra line break.

Just a few small copy edits for readability.

Reply:
@jorgeorpinel  Yeah Sure ,I'am getting used to with documentation structure and styling of DVC and hence making mistakes. 
Btw thanks, I will include these changes .

Reply:
These aren't mistakes per se, I'm just suggesting how this can be better. Looking fwd to your udpates.